doc/user/content/sql/create-sink.md

```diff
@@ -193,47 +193,62 @@ If the topic does not exist, Materialize will use the Kafka Admin API to create
 
 For Avro-encoded sinks, Materialize will publish the sink's Avro schema to the Confluent Schema Registry. Materialize will not publish schemas for JSON-encoded sinks.
 
-**Note:** With `reuse_topic` enabled, this schema for topic naming is ignored. Instead, the topic name specified in the sink definition is used as is.
-
 You can find the topic name for each Kafka sink by querying `mz_kafka_sinks`.
 
+{{< note >}}
 {{% kafka-sink-drop  %}}
+{{</ note >}}
 
-#### Enabling topic reuse after restart (exactly-once sinks)
+#### Exactly-once sinks (with topic reuse after restart)
 
 {{< beta />}}
 
 {{< version-added v0.9.0 />}}
 
-By default, Materialize creates new, distinct topics for sinks after each restart. To enable the reuse of the existing topic instead and provide exactly-once processing guarantees, Materialize must be able to do two things:
+By default, Materialize creates new, distinct topics for sinks on restart. To enable the reuse of an existing topic instead and achieve exactly-once processing guarantees, Materialize must:
```

Comment:
This intro could be rephrased to focus on EO guarantees, instead of topic reuse. If anyone would prefer that, I'll address it in the next iteration!

Reply:
I think we went back and forth on terminology a few times - the parameter was originally `exactly_once`, then switched to `reuse_topic`. My recollection from those discussions was that exactly_once is a well known term within industry, but without industry context reuse_topic better captured the end behavior. Now we seem to mix and match them a bit.

I have no preference and am quite willing to yield to your expertise if you think the focus on either eo or topic reuse would be clearer. 🙂

Reply:
It's definitely confusing! Users seem to ask more promptly about "exactly-once sinks" than "sink topic reuse", but it's not a clear cut. Let's see what feedback we get moving forward and adapt as/if needed!